lib/msf/ui/console/command_dispatcher/db.rb

```diff
@@ -100,21 +99,50 @@ def cmd_list_data_services()
           end
 
           def cmd_add_data_service(*args)
+            protocol = "http"
+            port = 80
+            https_opts = {}
             while (arg = args.shift)
               case arg
-                when '-h'
-                  host = args.shift
+                when '-h', '--help'
+                  cmd_add_data_service_help
+                  return
                 when '-p'
                   port = args.shift
+                when '-s', '--ssl'
+                  protocol = "https"
+                when '-c'
```

Comment:
Would be nice to include `--cert` to match `msfdb` options

Reply:
I had that, and then took it off. I don't know if this paradigm is used elsewhere in the app, but I was going with a certain style where if you pass a value after the flag then it would only have the `-letter` option. If the flag is just a setting or toggle, then it would have a `--word` option.

Does this make sense? Or is it a convention that I just created in my head?

Reply:
The `hosts` command has both `-letter` and `--word` options for options that take a parameter.
```
-S,--search       Search string to filter by
  -i,--info         Change the info of a host
```

Reply:
OK, so I guess I can add it back to conform with what the rest of the app is doing.